lib/wrappers/output.ts

```diff
@@ -0,0 +1,38 @@
+import fs, { PathLike } from "fs";
+import TransformNodeWrapper from "./transform-node";
+import { isAbsolute } from 'path';
+
+const logger = require('heimdalljs-logger')('broccoli:outputWrapper');
+
+const WHITELISTEDOPERATION = new Set([
+  'readFileSync',
+  'existsSync',
+  'lstatSync',
+  'readdirSync',
+  'statSync',
+  'writeFileSync',
+  'appendFileSync',
+  'rmdirSync',
+  'mkdirSync'
+]);
+
+export default function OutputWrapper (node: TransformNodeWrapper) {
+  return new Proxy(fs, {
+    get(target: any, propertyName: string): any {
+      return function() {
```

Comment:
This will return a new function **every time _any_** fs method is called (which we expect to be quite a lot). We should probably avoid doing extra work repeatedly...